packages/venia-ui/lib/components/FilterSidebar/filterSidebar.js

```diff
@@ -0,0 +1,145 @@
+import React, { useMemo, useCallback, useRef, Fragment } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { array, arrayOf, shape, string, number } from 'prop-types';
+import { useFilterModal } from '@magento/peregrine/lib/talons/FilterModal';
+
+import { mergeClasses } from '../../classify';
+import LinkButton from '../LinkButton';
+import CurrentFilters from '../FilterModal/CurrentFilters';
+import FilterBlock from '../FilterModal/filterBlock';
+import defaultClasses from './filterSidebar.css';
+
+const DEFAULT_FILTERS_OPEN_COUNT = 3;
+
+/**
+ * A view that displays applicable and applied filters.
+ *
+ * @param {Object} props.filters - filters to display
+ */
+const FilterSidebar = props => {
+    const { filters, filtersOpen } = props;
+    const talonProps = useFilterModal({ filters });
+    const {
+        filterApi,
+        filterItems,
+        filterNames,
+        filterState,
+        handleApply,
+        handleReset
+    } = talonProps;
+
+    const filterRef = useRef();
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const filtersToOpen =
+        typeof filtersOpen === 'number'
+            ? filtersOpen
+            : DEFAULT_FILTERS_OPEN_COUNT;
+
+    const handleApplyFilter = useCallback(
+        (...args) => {
+            const filterElement = filterRef.current;
+            if (
+                filterElement &&
+                typeof filterElement.getBoundingClientRect === 'function'
+            ) {
+                const filterTop = filterElement.getBoundingClientRect().top;
+                const offset = 150;
```

Comment:
It's the fixed header's height plus an additional offset to go just above the sorting button. If you'd like I can pass it in as a prop instead with a default of 150. Or I can create a ref in the parent component for the toolbar area, pass that as a prop, and use that ref instead of the filterElement

Reply:
I was hoping there would be some shared constant we could reference here (and in other components that need the header height). A prop/ref is unnecessary and according to @jimbo this may be "solved" in his incoming tailwind work.

At a minimum lets just move it to something above the function like `const HEADER_HEIGHT = 150;`